src/main.ml

```diff
@@ -11,6 +11,9 @@
 (*                                                                     *)
 (***********************************************************************)
 
+module Pervasives = struct
+  let exit = exit
+end
```

Comment:
If I understand, that's a very clever trick to write code that is compatible with both old OCaml versions (pre-Stdlib) and new OCaml versions. Right? If so, could you include a comment that says so? Because my first reaction was "let's just replace Pervasives.exit with Stdlib.exit!".

Reply:
good point. Done